pkg/webhooks/operator_webhooks.go

```diff
@@ -274,7 +277,7 @@ func (webhookConfig *IntegreatlyWebhookConfig) setupCerts(ctx context.Context, c
 
 	// Wait for the secret to te created
 	secret := &corev1.Secret{}
-	err = wait.PollImmediate(time.Second*1, time.Second*30, func() (bool, error) {
+	err = wait.PollUntilContextTimeout(context.TODO(), time.Second*1, time.Second*30, true, func(ctx2 context.Context) (bool, error) {
```

Comment:
ctx2?

Reply:
please see previouse comments. It looks good. Thank you